docs/content/_index.md

```diff
@@ -6,9 +6,31 @@ date: 2018-02-11T16:52:23-05:00
 <img src="/banner.png" width="600" alt="Athens Logo"/>
 
 
-## Welcome to Athens
+## Athens is a server for your Go packages
 
-Athens is the name of the combined project that includes a global registry for [Go Modules](https://github.com/golang/go/wiki/Modules) and a stand-alone proxy server that can be deployed on-premise to storage and control available Go modules for your organization.
+Welcome, Gophers! We're looking forward to introducing you to Athens.. 
+
+On this site, we document Athens in detail. We'll teach you what it does, why it matters, what you can do with it, and how you can run it yourself. Below is a brief summary for you.
+
+#### What does Athens do?
+
+Athens provides a server for [Go Modules](https://github.com/golang/go/wiki/Modules)  that you can run. It serves public code and your private code for you, so you don't have to pull directly from a version control system (VCS) like GitHub or GitLab
+
+#### Why does it matter?
```

Comment:
Maybe in a future PR, we should expand more on this section: talk about the vendor directory problems, authentication, blacklisting, HTTP vs git clone performance, as well as VCS/immutability.

Reply:
Were you thinking in a new section/doc page?

Reply:
@arschles either in this section of "why does it matter" or in a new page/section  like "motivation" or something. Basically, all the points I made above is usually *my* elevator pitch of when people ask me about why GoModules/Athens.

Reply:
how about a new page that we can do in a follow-up?

Reply:
I like the idea of a new page, keep this one short and we can link it here after it's written.

Reply:
cool. I created https://github.com/gomods/athens/issues/749 to follow-up on this